app/components/date-picker.js

```diff
@@ -1,19 +1,11 @@
+import DateInput from 'hospitalrun/components/date-input';
+
 // Dervied from http://spin.atomicobject.com/2013/10/29/ember-js-date-picker/
-import InputComponent from 'ember-rapid-forms/components/em-input';
-export default InputComponent.extend({
+export default DateInput.extend({
   dateSetAction: null, // Specify an action to fire when a date is selected
-  htmlComponent: 'date-input',
   minDate: null,
   maxDate: null,
   originalPropery: null,
   showTime: false,
-  yearRange: 10,
-
-  didReceiveAttrs(attrs) {
-    let dateProperty = this.get('property');
```

Comment:
I had to remove this because Ember now tracks if some property has been set more than once and throws an error.